### ISSUE TITLE
Another issue

### ISSUE THREAD

**Issue by HyperionTau** [zkeal/landchina_spider#3]
Hi Zkeal,
I met another issue these days. The spider worked quite well last week, but in this week, it keeps showing the following message and scraped no items. By the way, I am using the transaction1 to get land transaction data. This problem also appears in scraping the items from primary market. 

2018-07-18 09:56:21 [scrapy.log] INFO: []
2018-07-18 09:56:21 [scrapy.core.engine] DEBUG: Crawled (200) <POST http://www.landchina.com/default.aspx?tabid=264&ComName=default> (referer: http://www.landchina.com/default.aspx?tabid=264&ComName=default)
2018-07-18 09:56:31 [scrapy.log] INFO: []
2018-07-18 09:56:31 [scrapy.core.engine] DEBUG: Crawled (200) <POST http://www.landchina.com/default.aspx?tabid=264&ComName=default> (referer: http://www.landchina.com/default.aspx?tabid=264&ComName=default)
2018-07-18 09:56:43 [scrapy.log] INFO: []
2018-07-18 09:56:43 [scrapy.core.engine] DEBUG: Crawled (200) <POST http://www.landchina.com/default.aspx?tabid=264&ComName=default> (referer: http://www.landchina.com/default.aspx?tabid=264&ComName=default)
2018-07-18 09:56:47 [scrapy.extensions.logstats] INFO: Crawled 9 pages (at 7 pages/min), scraped 0 items (at 0 items/min)

Now, I have no idea how to modify this code. Could you please help me on that?

**Comment by zkeal** [comment]
Hi HyperionTau,
I got your question,Could you visit landchina directly?I am not sure whether your ip are banned by landchina?

In fact LandChina has many strict, often changing anti-spider strategies but I'd be happy to help you.

**Comment by HyperionTau** [comment]
Yes, I can directly access the website. But it is not stable. Maybe the reason is that I am not in mainland China now. 
Never mind. This program worked well two weeks ago in Hong Kong. 


发送自 Windows 10 版邮件应用

发件人: zkeal
发送时间: 2018年7月18日 10:34
收件人: zkeal/landchina_spider
抄送: HyperionTau; Author
主题: Re: [zkeal/landchina_spider] Another issue (#3)

Hi HyperionTau,
I got your question,Could you visit landchina directly?I am not sure whether your ip are banned by landchina?
In fact LandChina has many strict, often changing anti-spider strategies but I'd be happy to help you.
